pkgs/applications/networking/p2p/ktorrent/default.nix

```diff
@@ -22,6 +22,14 @@ stdenv.mkDerivation rec {
     libktorrent taglib libgcrypt kplotting
   ];
 
+  patches = [
+    # Fix build with CMake 3.11
+    (fetchpatch {
+      url = "https://cgit.kde.org/ktorrent.git/patch/?id=672c5076de7e3a526d9bdbb484a69e9386bc49f8";
+      sha256 = "10pvs8wnlv1f1bgd3zys40a6lrrw2m0swancv7gj4pwpb9p28qwy";
```

Comment:
incorrect hash sum:

```
fixed-output derivation produced path '/nix/store/dn9w2hdcwcihwdj62ymwprxfn0nq73li-?id=672c5076de7e3a526d9bdbb484a69e9386bc49f8' with sha256 hash '1cn4rnbhadrsxqx50fawpd747azskavbjraygr6s11rh1wbfrxid' instead of the expected hash '10pvs8wnlv1f1bgd3zys40a6lrrw2m0swancv7gj4pwpb9p28qwy'
```